examples/user_defined_iface.yml

```diff
@@ -0,0 +1,43 @@
+---
+user-defined:
+    interface-types:
```

Comment:
What's the reason for the extra level? Adding `type: interface` as a property to the items would be an alternative to declare that this is a custom interface type.

Reply:
Reserve for future `user-defined` stuff.

Reply:
You can still define other plugins with this as well by using different types:

```
plugins:
    - type: interface
      name: vxcan
      handler: ....
   - type: routes
     name: blackhole
     handler: ...
```